### ISSUE TITLE
App name too long for share sheet

### ISSUE THREAD

**Issue by gingerbeardman** [acez/bookmark-companion#30]
- I guess it could just say "Bookmarks"?

![image](https://github.com/acez/bookmark-companion/assets/49612/250c12df-cb5d-4127-8a1a-eefeae1efe6d)

![image](https://github.com/acez/bookmark-companion/assets/49612/2d30ba0a-61a6-4916-bf57-6f7608982c75)

**Comment by acez** [comment]
You're right - didn't pay attention to that line break there. But not sure if it is possible to update the display name of the share extension. I will check that out. 
Not sure about just using bookmarks as there might be other bookmarking apps installed on a users device. But maybe (when it's possible) just adding a space between Bookmark and Companion could break the app name to make it look better. Will test that out, thanks for mentioning it.

**Comment by gingerbeardman** [comment]
I think once people are aware of the app icon, the text can say anything?

Or you could rename to something shorter? Bookmark Buddy? Bookmarker?

I don't know :)
